vscode-lean4/src/utils/localStorage.ts

```diff
@@ -11,7 +11,7 @@ export class LocalStorageService {
 
     getLeanPath() : string
     {
-        return this.storage.get<string>('LeanPath', null);
+        return this.storage.get<string>('LeanPath', '');
```

Comment:
This took me a second to appreciate.  `''` is falsy in javascript, so all the `if (leanPath) ...` checks take the same path as before.

I'm not sure if `''` is the best value to indicate that no version has been set; but at least it typechecks now.

Reply:
yeah, I think at least it is consistent now, all string properties we get this way default to empty string.